### ISSUE TITLE
Content root not respected while copying template

### ISSUE THREAD

**Issue by jacobsenj** [enriquepiatti/Magicento#81]
When using a local project and handling files on a network share via the directories functionality in PhpStorm project settings the copy template function will recognize package and themes on the network location correctly:

![bildschirmfoto 2013-08-12 um 12 21 33](https://f.cloud.github.com/assets/399303/946691/21a13a68-033a-11e3-8c3d-ce997e5bc8d8.png)

but then copies the template to the local project folder instead of using the corresponding content root setting:

![bildschirmfoto 2013-08-12 um 12 21 56](https://f.cloud.github.com/assets/399303/946698/42011030-033a-11e3-8fb8-1e663958c970.png)

![bildschirmfoto 2013-08-12 um 12 23 24](https://f.cloud.github.com/assets/399303/946692/2aea438a-033a-11e3-8364-f7e189769f86.png)

Local path is ~/Projects/SMG in this case. In here there's only the .idea folder with the PhpStorm project files. The source files are on a network drive located here:
![bildschirmfoto 2013-08-12 um 12 24 01](https://f.cloud.github.com/assets/399303/946689/fd2dd236-0339-11e3-9802-b584eaf7220c.png)

**Comment by bgalileo** [comment]
I'm having the same problem, this is extremely annoying.

**Comment by enriquepiatti** [comment]
I think I don't understand very well how are you managing your project.
You have created a project in PhpStorm located in your hdd and then removed the content root from there and added a new Content Root pointing to some network?
Is there any special reason for doing that instead of using a version contorl system, or configuring a deployment to send the changes automatically to that network?
Anyway, if you can give me more details about your project settings/workflow maybe I can take a look at this, I really never used something similar to that in PhpStorm.

**Comment by bgalileo** [comment]
> You have created a project in PhpStorm located in your hdd and then removed the content root from there and added a new Content Root pointing to some network?

Yep. I'm accessing the document root of my apache server via a network drive. Creating the project in a local folder and changing the Content Root afterwards is the only way to prevent PHPStorm from creating the .idea folder on the network drive.

**Comment by bgalileo** [comment]
By the way, sometimes Magicento creates files in the correct folders. IIRC, only copying template files and creating new modules doesn't work.

**Comment by enriquepiatti** [comment]
I will try to check it.
Why are you using that workflow anyway? is not slowing down your IDE having the code in a network?   if is not possible to use a VCS (git, etc), have you tried using a deployment setting to upload your local file changes automatically to the Network?

**Comment by bgalileo** [comment]
I know our current workflow is not perfect, but so far we have found no alternative our designers are comfortable with. They're editing the files on the development server and will continue to do so. Since the designers are the majority in our company, a development-centric workflow (like using local git repositories) would hurt productivity.

**Comment by jacobsenj** [comment]
The project setup would be
- create an empty project locally (~/Projects/SMG in my example above)
- use the directories functionality in the project settings to "mount" a network resource into the project and remove the default directory which has been added upon creation of the empty project (/Volumes/www/smg/jeja.smg.dev in my example above)

We had to use this workflow because our designers are not quite git-compatible and not everybody has a local development environment to work with, so we have a development server where the work can be done. And of course PHPStorm won't let you open a network located directory as new project.
